artifact-management/commands/gem-test.yml

```diff
@@ -1,36 +1,16 @@
-description: Perform the actual gem build
+description: Perform simple gem test
 parameters:
-  bundler_version:
```

Comment:
Here you've removed my ability to specify the `bundler_version`, I think. I realize it's part of the new `gem-test-common-steps` but as a user I'm not going to call `gem-test-common-steps` directly, I'm probably going to call `gem-test` or `gem-test-multiple-ruby-versions`. I want to call it like this:

```
- gem-test:
  bundler_version: 1.17.4
```

and I expect the `bundler_version` to be used wherever it is needed (in this implementation, it would be passed into the `gem-test-common-steps` command. But you've taken that ability away from me by removing this parameter here.